SimG4Core/Geometry/src/DDG4ProductionCuts.cc

```diff
@@ -113,9 +113,9 @@ void DDG4ProductionCuts::dd4hepInitialize() {
 
   for (auto const& it : *dd4hepMap_) {
     for (auto const& fit : specs) {
-      for (auto const& pit : fit->paths) {
+      for (auto const& pit : fit.second->paths) {
         if (dd4hep::dd::compareEqual(dd4hep::dd::noNamespace(it.first.name()), dd4hep::dd::realTopName(pit))) {
```

Comment:
@ianna Why not dd4hep::dd::compareEqualName here as well? :)
There could be regex in ProdCuts as well?

Reply:
@ghugo83 - well spotted! Thanks.

Reply:
done in https://github.com/cms-sw/cmssw/pull/31915